libr/asm/arch/arm/aarch64/aarch64-opc.c

```diff
@@ -2223,7 +2223,11 @@ static void
 print_register_offset_address (char *buf, size_t size,
 			       const aarch64_opnd_info *opnd)
 {
+#ifdef _MSC_VER
```

Comment:
just use the define for msvc and gcc. no need to dupe this case. in fact there's no need to even use a define, just hardcode 16 in the tblen